src/cli/index.ts

```diff
@@ -90,8 +91,37 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
     .option("--dry-run", "dry run (skip actually publishing)")
     .option("-d, --debug", "show debug logs")
     .option("--ci", "Continuous Integration mode")
+    .option(
+      "-n, --name [name]",
+      "Name of the pre-built or released contract (such as nft-drop)",
+    )
+    .option(
+      "--cv, --contract-version [version]",
+      "Version of the released contract",
+    )
     .action(async (options) => {
-      const url = await processProject(options, "deploy");
+      let url;
+
+      if (options.name) {
+        url = generateDashboardUrl(options.name, options.contractVersion);
+
+        if (!url) {
+          logger.error(
+            chalk.red(
+              `Could not find a contract named ${options.name} ${
+                options.contractVersion || ""
+              }`,
+            ),
+          );
+          return;
+        }
+
+        info(`Open this link to deploy your contract:`);
+        logger.info(chalk.blueBright(url));
```

Comment:
This should also open the browser automatically like we do for deploy/release

Reply:
not sure what you mean? release/deploy doesn't open automatically for me

Reply:
might be a windows thing, but look at what we do in`processor.ts` - we call `open(url)` which attempts to open your default browser